apps/src/templates/teacherHomepage/ManageSectionsCollapsible.story.jsx

```diff
@@ -39,6 +39,7 @@ export default storybook => {
                 sectionCode: "HPRWHG"
               },
             ]}
+            codeOrgUrlPrefix = "http://localhost:3000/"
```

Comment:
I don't know the right solution for storybook, but this feels more hardcoded we'd usually like to use.  And doesn't take advantage of our `localhost.code.org` capability.  Have we had a need for this kind of ability in a storybook anywhere else?